main.go

```diff
@@ -778,7 +778,8 @@ func main() {
 	hsIndex = hsClient.InitIndex(conf.Hearthstone.IndexName)
 
 	// Initialise Battlenet client
-	bNetClient = blizzard.NewClient(conf.BattleNet.ClientID, conf.BattleNet.ClientSecret, blizzard.US, blizzard.EnUS)
+	if (conf.BattleNet.ClientSecret != "") {
+		bNetClient = blizzard.NewClient(conf.BattleNet.ClientID, conf.BattleNet.ClientSecret, blizzard.US, blizzard.EnUS)
```

Comment:
Something weird going on with the indenting

Reply:
Clarify? Looks like 8-space tabs to me.